js/ajax_settings.js

```diff
@@ -43,6 +45,7 @@ onOffice.ajaxSaver = function(outerDiv) {
 			var newKey = mergeElement[i];
 			data[newKey] = onOffice.settings[newKey];
 		}
+		$('#onoffice-notice-wrapper').empty();
 
 		jQuery.post(onOffice.settings.ajax_url, data, function(response) {
```

Comment:
I'm just noticing that we are still doing an AJAX post. The idea behind this PR is to not use the async JS POST, but the normal form post that is common for WordPress. This is what I meant with "reload edit pages after saving", sorry if that wasn't clear!

Is the difference clearer now or do you have questions? We can maybe do this change in another PR, and focus on the copy button in this one.

Reply:
I used the ajax call like creating a WordPress post. They will use the replaceState to change the url. I will update this to normal form post if you think it will be better.